packages/frontend/app/components/global-search.js

```diff
@@ -108,13 +123,4 @@ export default class GlobalSearchComponent extends Component {
     this.args.onSelectPage(1);
     this.args.setIgnoredSchoolIds(ignoredSchoolIds);
   }
-
-  search = restartableTask(async (el, [query]) => {
```

Comment:
this is incomplete. the component's template is supposed to renders a spinner while the search task is running.

![image](https://github.com/ilios/frontend/assets/1410427/152e3a13-c488-46a7-be5f-ac475fa205ff)

it now displays `No results` instead while search is running. 

![image](https://github.com/ilios/frontend/assets/1410427/c1b694b4-5996-465a-ae69-493d102f132d)

Reply:
Fixed, added a test.